spring-cloud-generator/generate-one.sh

```diff
@@ -73,40 +73,45 @@ bazel build //$googleapis_folder:"$client_lib_name"_java_gapic_spring
 cd -
 
 ## copy spring code to outside
-mkdir -p ../generated
-cp googleapis/bazel-bin/$googleapis_folder/"$client_lib_name"_java_gapic_spring-spring.srcjar ../generated
+mkdir -p ../spring-cloud-previews
+cp googleapis/bazel-bin/google/cloud/$client_lib_name/v1/"$client_lib_name"_java_gapic_spring-spring.srcjar ../spring-cloud-previews
```

Comment:
This line might have gotten changed unintentionally? 
```suggestion
cp googleapis/bazel-bin/$googleapis_folder/"$client_lib_name"_java_gapic_spring-spring.srcjar ../spring-cloud-previews
```

Reply:
Great catch! this almost slipped in.

Reply:
fixed in [update](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1394/commits/035e302400e813a8db39eae46e7a0eb13fd92ede)